applications/mne_analyze/plugins/averaging/averaging.cpp

```diff
@@ -0,0 +1,633 @@
+//=============================================================================================================
+/**
+ * @file     averaging.cpp
+ * @author   Gabriel Motta <gbmotta@mgh.harvard.edu>
+ * @since    0.1.2
```

Comment:
this should be 0.1.3 please check all other cpp and header files, which you created in this PR, as well